cartography/intel/aws/redshift.py

```diff
@@ -77,9 +78,10 @@ def load_redshift_cluster_data(neo4j_session, clusters, region, current_aws_acco
             aws_update_tag=aws_update_tag,
         )
         _attach_ec2_security_groups(neo4j_session, cluster, aws_update_tag)
+        _attach_iam_roles(neo4j_session, cluster, aws_update_tag)
+        _attach_aws_vpc(neo4j_session, cluster, aws_update_tag)
 
 
-@timeit
```

Comment:
I actually like having a timer on each individual method so you know which ones are slow.  That being said if the overall latency is really low then it might not be necessary